src/browser/sidebar.js

```diff
@@ -107,3 +108,13 @@ export const view = ({entries}, address) =>
       className: 'sidebar-toolbar'
     })
   ]);
+
```

Comment:
@Gozala that's not going to work when we start animating between modes. Transitions are more complex than just hide/show. We need to be able to modify the style of the root view as it transitions.